web/src/main/java/com/epam/rft/atsy/web/exceptionhandling/UncheckedExceptionResolver.java

```diff
@@ -35,7 +51,19 @@ public ModelAndView resolveException(HttpServletRequest httpServletRequest,
       return PASS_TO_NEXT_RESOLVER;
     }
 
-    httpServletResponse.setStatus(HttpServletResponse.SC_INTERNAL_SERVER_ERROR);
+    int httpStatusCode =
+        AnnotationUtils.isAnnotationDeclaredLocally(ResponseStatus.class, e.getClass())
+            ? AnnotationUtils.findAnnotation(e.getClass(), ResponseStatus.class).value().value()
```

Comment:
ResponseStatusExceptionResolver uses `code().value()`. Please consider to change to that method.